Added media message components

ajaman190 opened the pull request:
**Description**
This pull request addresses the issue: "Create photo, sticker, video, voice, document, and location message component."

**Changes Made**
- Implemented the photo, video thumbnail, sticker, location and document message component.
- Partially implemented the audio player component. Further improvements are needed to make it fully functional.

**Things Left**
- Implementation of photo viewer and video player.

**Screenshots**

https://github.com/glific/mobile/assets/84832565/2960887a-595b-4f49-9a57-71f378478c80



**Closes Issue**
Closes: #89 

@mdshamoon @VishalZ123 Review the changes and provide feedback.

mdshamoon reviewed (approved):
Looks good but our test coverage is slipping. Let's ensure that we add tests as well.

ajaman190 commented:
** Changes made**
- Added video player
- Added chat screen tests, dropped options tab animation and fixed multiple state tabs issue
- Fixed incoming test & lint issues from the upstream

**Screenshots**

https://github.com/glific/mobile/assets/84832565/5e802002-7c0a-4b90-af03-925d845ae336

**Closes Issue**
Closes: #98

ajaman190 commented:
@mdshamoon Most of the deep scan issues have been fixed by @VishalZ123 in this PR #97, as they were coming from upstream. So, I left them.
